packages/salesforcedx-vscode-soql/package.json

```diff
@@ -94,8 +94,8 @@
   "contributes": {
     "customEditors": [
       {
-        "viewType": "%soqlCustom_soql%",
-        "displayName": "SOQL Builder",
+        "viewType": "soqlCustom.soql",
+        "displayName": "%soqlCustom_soql%",
```

Comment:
Are the `%` symbols for i18n?

Reply:
Yeah, these look up the values in package-i18n.json